README.md

```diff
@@ -1,32 +1,75 @@
 # Group 2 - Unstructured Data [![Join the chat at https://gitter.im/MusicConnectionMachine/UnstructuredData](https://badges.gitter.im/MusicConnectionMachine/UnstructuredData.svg)](https://gitter.im/MusicConnectionMachine/UnstructuredData?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/488966f28a0c448cac974baa104b74cc)](https://www.codacy.com/app/kordianbruck/UnstructuredData?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=MusicConnectionMachine/UnstructuredData&amp;utm_campaign=Badge_Grade)
+This project aims to filter for entities in unstructured data using data from the [Common Crawl](http://commoncrawl.org/).
+- This project pulls it's entities from a PostgreSQL database,
+- Pulls data from the Common Crawl and simultaneously filters for entities,
+- And then saves the found entities back in the DB and uploads relevant pages into an [Azure](https://azure.microsoft.com) blob storage container.
+
+This project allows for distributed computing as all tasks are being handled by an Azure task queue.
+
 
 ## Fetch dependencies and compile source
 Navigate to the project root and run `yarn install`. [Yarn](https://yarnpkg.com/lang/en/) has to be installed on the machine.  
 Once that's finished run `npm run compile` to compile everything to Javascript.
 
 ## Usage
-Via command line interface:
 ```
-$ node out/app.js -h
-
-> Usage: app [options]
+$ Usage: app [options]
 
-  Options:
+> Options:
 
     -h, --help                                 output usage information
+    -P, --Process                              process queue items
```

Comment:
Using both `-A` and `-P` would replicate the functionality of the version where we didn't have the Azure queue.